.github/ISSUE_TEMPLATE/rubber-duck.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Rubber Duck Template 🦆
+about: Use for students needing to think things out
+title: ''
+labels: help wanted
+assignees: ''
+---
+
+##### _Incomplete tickets will not be evaluated._
+
+---
+
+#### Your Name:
+
+#### What are you trying to do?
+
+#### What is preventing you from doing it?
+
+#### What have you googled? (Paste links to the things you have found from your google search - You must have at least 3)
```

Comment:
@jacobpaine do you think this is necessary for the rubber duck? I was thinking that this was more or less trying to get someone to jump in and chat a bit without too much work to let us know. Thoughts?

Reply:
Hmmm agreed. I was probably considering a problem that was too specific, e.g. I can't think of idea for capstone.

Reply:
Ah ok. Yeah. I guess the conversation template could cover that too.